inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx

```diff
@@ -178,11 +184,14 @@ const Comp = ({ inlongGroupId, readonly, mqType }: Props, ref) => {
             <Button type="link" onClick={() => onDelete(record)}>
               {t('basic.Delete')}
             </Button>
-            {record?.status && (record?.status === 120 || record?.status === 130) && (
-              <Button type="link" onClick={() => onWorkflow(record)}>
-                {t('meta.Stream.ExecuteWorkflow')}
-              </Button>
-            )}
+            {record?.status &&
+              ((groupStatus === 130 && record?.status === 100) ||
```

Comment:
Are the two conditions `groupStatus === 130 && record?.status === 100` and `groupStatus === 130` repeated, and only need `groupStatus === 130`?